packages/taiko-client/driver/chain_syncer/blob/blocks_inserter/common.go

```diff
@@ -93,7 +118,11 @@ func createExecutionPayloads(
 		return nil, err
 	}
 
-	fc := &engine.ForkchoiceStateV1{HeadBlockHash: meta.ParentHash}
+	fc := &engine.ForkchoiceStateV1{
+		HeadBlockHash:      meta.ParentHash,
+		SafeBlockHash:      lastVerfiiedBlockHash,
```

Comment:
Why add `SafeBlockHash` and `FinalizedBlockHash`?

Reply:
Think its a nice to have?

Reply:
I don't think so, it's an unnecessary addition.

Reply:
changed: https://github.com/taikoxyz/taiko-mono/pull/18864/commits/15eb9be7a23b321b9254168dc3bb5b511113a518